doc/apps/s_client.pod

```diff
@@ -420,7 +420,7 @@ command for more information.
 send the protocol-specific message(s) to switch to TLS for communication.
 B<protocol> is a keyword for the intended protocol.  Currently, the only
 supported keywords are "smtp", "pop3", "imap", "ftp", "xmpp", "xmpp-server",
-"irc" and "postgres."
+"irc", "postgres", and "ldap".
```

Comment:
I agree with @akostadinov ...  having the period inside the quotes may make it confusing to know if it's part of the keyword or not.  Having it outside the quotes makes that clear.